the-collab-lab/tcl-34-smart-shopping-list
# Welcome view

## Description
We created styles for the Welcome view to be able to access from the mobile or laptop and still see the application correctly without style breaks.

## Related Issue

#32 

## Acceptance Criteria
* El usuario quiere poder ver la aplicación desde su celular y también desde su laptop sin afectarse el diseño de la aplicación
* Dependiendo del tamaño que tenga puede variar el diseño de la vista welcome

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓| :nail_care: Styles           |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/42523734/142294775-5761984c-06c5-4d65-ae89-a69c3201e47e.png)


### After

![image](https://user-images.githubusercontent.com/42523734/142732808-4d3d541a-2585-4756-8e0f-f58511fe6e96.png)
![image](https://user-images.githubusercontent.com/42523734/142732825-53b6cc92-eebd-4410-9040-9e8f2d1477af.png)



## Testing Steps / QA Criteria

* From your terminal, pull down this branch with ```git pull welcome-view``` and check that branch out with ```git switch welcome-view```
* ```npm start``` to launch the app.
